csrc/nv_internal/cpp/kernels/quantization.cu

```diff
@@ -181,7 +181,8 @@ void invokeFP4Quantization(int m, int n, T const* input, float const* SFScale, i
 template <typename T, int SF_VEC_SIZE>
 void invokeBatchedFP4Quantization(int b, int m, int n, T const* input, float const* SFScale,
                                   int64_t* output, int32_t* SFOuput, bool useUE8M0,
-                                  int multiProcessorCount, cudaStream_t stream) {
+                                  int multiProcessorCount, cudaStream_t stream,
+                                  FP4QuantizationSFLayout layout) {
```

Comment:
We usually place `cudaStream_t stream` as the last argument per our programming convention.